tasks/post_copy_request_to_queue/test/unit_tests/test_post_copy_request_to_queue.py

```diff
@@ -1 +1,318 @@
-test
+"""
+Name: test_post_copy_request_to_queue.py
+Description: unit tests for post_copy_request_to_queue.py 
+
+"""
+import boto3
+import json
+import os
+import random
+from moto import mock_secretsmanager
+from post_copy_request_to_queue import handler, task, exponential_delay
+from unittest import mock, TestCase
+from unittest.mock import Mock, call, patch, MagicMock
+from requests_db import get_dbconnect_info, DatabaseError
+from database import single_query, result_to_json
+from botocore.exceptions import ClientError
+
+class TestPostCopyRequestToQueue(TestCase):
+    """
+    Unit tests for the post_copy_request_to_queue lambda function.
+    """
+    # Create the mock instance of secrets manager
+    mock_sm = mock_secretsmanager()
```

Comment:
Not sure why you are mocking secrets manager here. You would probably want to mock SQS. 

Looking at your setup, you would also want to mock `get_dbconnect_info` in the tests that require it.

Reply:
removed secrets manager and added mock get_dbconnect_info